segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/operations/MergeSegmentOperation.java

```diff
@@ -148,12 +183,23 @@ protected void beforeSerialization(MergeSegmentOperation o) {
         }
 
         private void write00(MergeSegmentOperation o, RevisionDataOutput target) throws IOException {
-            target.length(SERIALIZATION_LENGTH);
+            int attributesLength = o.attributeUpdates == null ? target.getCompactIntLength(0) :
```

Comment:
This will break backwards compatibility. Please declare a new revision and ONLY write the attributes in that new revision (and read them from there). Leave revision 0 as it was before.

Reply:
True, my fault. I added new write (`write01`) and read (`read00`) methods for the new operation version, as well as the new revision declaration.